### PR TITLE
Allow developers to set the JWT auth token expiration period

### PR DESCRIPTION
**Pull request by arnaldopereira**
This change is [Reviewable](https://reviewable.io/reviews/luizalabs/teresa/354)

### CODE REVIEW

File: pkg/server/user/handlers.go [hunk 1]
```diff
@@ -14,7 +16,12 @@ type Service struct {
 }
 
 func (s *Service) Login(ctx context.Context, request *userpb.LoginRequest) (*userpb.LoginResponse, error) {
-	token, err := s.ops.Login(request.Email, request.Password)
+	exp := time.Hour * 24 * 15
+	e := ctx.Value("expires_in")
```
Comment (yagonobre) [inline pkg/server/user/handlers.go]: Maybe get this from `request`

Comment (arnaldopereira) [reply pkg/server/user/handlers.go]: Done, thanks

File: pkg/client/cmd/login_cluster.go [hunk 2]
```diff
@@ -30,6 +37,7 @@ func login(cmd *cobra.Command, args []string) {
 		cmd.Usage()
 		return
 	}
+	fmt.Printf("expiresin: #v float: %f\n", expiresIn, float64(expiresIn))
```
Comment (drgarcia1986) [inline pkg/client/cmd/login_cluster.go]: Could you remove this print? (Is it a debug print?)

File: pkg/client/cmd/login_cluster.go [hunk 3]
```diff
@@ -12,15 +15,19 @@ import (
 )
 
 var userName string
+var expiresIn time.Duration
```
Comment (drgarcia1986) [inline pkg/client/cmd/login_cluster.go]: What about?:
```diff
- var userName string
- var expiresIn time.Duration
+ var (
+   userName string 
+   expiresIn time.Duration
+ )
```

File: pkg/client/cmd/login_cluster.go (hunk 2 above)
Comment (arnaldopereira) [reply pkg/client/cmd/login_cluster.go]: 😮 
removed

File: pkg/client/cmd/login_cluster.go (hunk 3 above)
Comment (arnaldopereira) [reply pkg/client/cmd/login_cluster.go]: done

**Pull request merged**